contrib/python/README.md

```diff
@@ -7,9 +7,33 @@ See [libpod](https://github.com/projectatomic/libpod)
 
 ###
 
-To build the podman wheel:
+To build the podman egg:
```

Comment:
Is egg really a term used to name this type of utility?

Reply:
> Is egg really a term used to name this type of utility?

[Yup][1], I'd guess because pythons lay eggs ;).

[1]: https://wiki.python.org/moin/egg

Reply:
Thanks for the pointer.  I'd thought "jar" like files in python were named something other than eggs, but I guess not.  So many languages, so little time.